providers/shared/components/component.ftl

```diff
@@ -965,6 +969,480 @@
+    },
+    {
+        "Names" : "LogDriver",
+        "Type" : STRING_TYPE,
+        "Values" : ["awslogs", "json-file", "fluentd"],
+        "Default" : "awslogs"
+    },
+    {
+        "Names" : "VolumeDrivers",
+        "Type" : ARRAY_OF_STRING_TYPE,
+        "Values" : [ "ebs" ],
+        "Default" : []
+    },
+    {
+        "Names" : "ClusterLogGroup",
+        "Type" : BOOLEAN_TYPE,
+        "Default" : true
+    },
+    {
+        "Names" : "Links",
+        "Subobjects" : true,
+        "Children" : linkChildrenConfiguration
+    },
+    {
+        "Names" : "Profiles",
+        "Children" :
+            [
+                {
+                    "Names" : "Processor",
+                    "Type" : STRING_TYPE,
+                    "Default" : "default"
+                },
+                {
+                    "Names" : "Alert",
+                    "Type" : STRING_TYPE,
+                    "Default" : "default"
+                },
+                {
+                    "Names" : "Network",
+                    "Type" : STRING_TYPE,
+                    "Default" : "default"
+                },
+                {
+                    "Names" : "Logging",
+                    "Type" : STRING_TYPE,
+                    "Default" : "default"
+                }
+            ]
+    },
+    {
+        "Names" : "AutoScaling",
+        "Children" : autoScalingChildConfiguration
+    },
+    {
+        "Names" : "HostScalingPolicies",
+        "Subobjects" : true,
+        "Children" : scalingPolicyChildrenConfiguration
+    },
+    {
+        "Names" : "DockerUsers",
+        "Subobjects" : true,
+        "Children" : [
+            {
+                "Names" : "UserName",
+                "Type" : STRING_TYPE
+            },
+            {
+                "Names" : "UID",
+                "Type" : NUMBER_TYPE,
+                "Mandatory" : true
+            }
+        ]
+    },
+    {
+        "Names" : "LogMetrics",
+        "Subobjects" : true,
+        "Children" : logMetricChildrenConfiguration
+    },
+    {
+        "Names" : "Alerts",
+        "Subobjects" : true,
+        "Children" : alertChildrenConfiguration
+    },
+    {
+        "Names" : "Hibernate",
+        "Children" : [
+            {
+                "Names" : "Enabled",
+                "Type" : BOOLEAN_TYPE,
+                "Default" : false
+            },
+            {
+                "Names" : "StartUpMode",
+                "Type" : STRING_TYPE,
+                "Values" : ["replace"],
+                "Default" : "replace"
+            }
+        ]
+    },
+    {
+        "Names" : "Role",
+        "Description" : "Server configuration role",
+        "Default" : ""
+    },
+    {
+        "Names" : "ComputeProvider",
+        "Description" : "The default compute provider for the cluster",
+        "Values" : [ "ec2", "ec2OnDemand" ],
+        "Default" : "ec2"
+    }
+]]
+
+[#assign containerServiceAttributes = [
+    {
+        "Names" : "Engine",
```

Comment:
I think this should be captured via links not on the component. To switch from fargate to ECS implies the creation of a cluster - something that should be managed on the container host not the service.

Reply:
Updated in fae79f6  and 5cdc74c